app/jobs/score_updater_job.rb

```diff
@@ -0,0 +1,22 @@
+class ScoreUpdaterJob
+  include SuckerPunch::Job
+
+  def perform(user_id, score=1)
```

Comment:
@dalpo @bugant if you don't want to call it *delta*, let's call it *amount*, that is the same name they're using in the mongodb official guide: https://docs.mongodb.com/manual/reference/operator/update/inc/.